app/controllers/requests_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RequestsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @request = book.requests.create(
+                request_params.merge(user: current_user, status: 'pending'))
```

Comment:
you can set a default in the database for the status to pending, so you don't need to specify the pending status always that is created one request.